api/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -188,6 +203,64 @@ func (in *ContentSource) DeepCopyObject() runtime.Object {
 	return nil
 }
 
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
+func (in *ContentSourceBinding) DeepCopyInto(out *ContentSourceBinding) {
+	*out = *in
+	out.TypeMeta = in.TypeMeta
+	in.ObjectMeta.DeepCopyInto(&out.ObjectMeta)
+	out.ClassRef = in.ClassRef
```

Comment:
References to a ClassRef in ContentSourceBinding?. seems like stale code

Reply:
Correct. Thanks for catching this. I'll send an MR to correct this.

Reply:
Fixed it here: https://github.com/vmware-tanzu/vm-operator-api/pull/32/